test/index.js

```diff
@@ -0,0 +1,61 @@
+const seleniumAssistant = require('selenium-assistant');
+const serve = require('serve');
+const path = require('path');
+const Mocha = require('mocha');
+const chai = require('chai');
+
+async function initTest() {
+  console.log('downloading browsers...');
+  const expiration = 24;
+  await seleniumAssistant.downloadLocalBrowser('chrome', 'stable', expiration);
+
+  console.log('starting server...');
+  const serveDir = path.join(__dirname, '../');
+  server = serve(serveDir, {
+      port: 6881,
+  });
+
+  const browsers = seleniumAssistant.getLocalBrowsers();
+  browsers.forEach((browser) => {
+    // Skip if the browser isn't stable.
+    if (browser.getReleaseName() !== 'stable') {
+      return;
+    }
+    // Print out the browsers name.
+    if(browser.getPrettyName() !== "Google Chrome Stable") {
+        return;
```

Comment:
Let's use `prettier` to avoid style conflicts when we paste code in from other repos.

Reply:
done